GVFS/GVFS.Installer.Mac/scripts/Distribution_Meta.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <pkg-ref id="com.git.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+            <line choice="com.git.pkg"/>
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="0" onConclusion="none">VFSForGit.0.2.173.2.pkg</pkg-ref>
+    <choice id="com.git.pkg" visible="false">
+        <pkg-ref id="com.git.pkg"/>
+    </choice>
+    <pkg-ref id="com.git.pkg" version="2.20.1.vfs.1.1.102.gdb3f8ae2d1" onConclusion="none">git-2.20.1.vfs.1.1.102.gdb3f8ae2d1-intel-universal-snow-leopard.pkg</pkg-ref>
+    <installation-check script="InstallationCheck()"/>
```

Comment:
How does this XML file work?  Is `InstallationCheck()` a special well known function?

Reply:
How this XML file is used by the tools during packaging is not very well documented, there is a reference for its scheme [here](https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/DistributionDefinitionRef/Chapters/Distribution_XML_Ref.html). In our case, I used `productbuild --synthesize` command which reads constituent packages info and automatically creates Distribution.xml. I manually then replaced version numbers and package names with placeholders. Our build script will substitute the place holders with actual dynamic values during build time.

About `InstallationCheck()` - any javascript function can be specified in `<installation-check>`, naming is not important. [This page](https://matthew-brett.github.io/docosx/flat_packages.html) has some info on sample `InstallationCheck()` implementation.